crates/rattler/src/install/mod.rs

```diff
@@ -876,4 +905,40 @@ mod test {
 
         insta::assert_yaml_snapshot!(paths);
     }
+
+    #[tracing_test::traced_test]
+    #[tokio::test]
+    async fn test_link_package_from_archive() {
+        let packages = vec![(
+            "https://conda.anaconda.org/conda-forge/win-64/ruff-0.0.171-py310h298983d_0.conda",
+            "25c755b97189ee066576b4ae3999d5e7ff4406d236b984742194e63941838dcd"
+        ),
+        (
+            "https://conda.anaconda.org/conda-forge/linux-aarch64/bzip2-1.0.8-hf897c2e_4.tar.bz2",
```

Comment:
I think this package contains symlinks which won't work on windows. Can you use another package?

Reply:
Or if possible a smaller package stored in the repo? Premature opimization perhaps, but I want the tests to run very fast.

Reply:
[`48d9c20` (#937)](https://github.com/conda/rattler/pull/937/commits/48d9c204e093e896919120c5a229fd3385688318)